docs/platform/concepts/user-access-management.md

```diff
@@ -4,8 +4,8 @@ title: User and access management
 
 There are several types of users in the Aiven Platform:
 
-- [Super admin](/docs/platform/howto/make-super-admin): Users with full access to the
-  organization and are not subject to controls like authentication policies.
+- Super admin: Users with full access to the organization and are not subject to
```

Comment:
We should remove "are not subject to controls like authentication policies". Right now they are but we will change that soon hopefully.